### ISSUE TITLE
Bug: wrong path in sharing dialog

### ISSUE THREAD

**Issue by AuspeXeu** [mattermost/focalboard#3707]
## Steps to reproduce the behavior

1. In you settings .json, add the following line `"serverRoot": "https://example.com/vhost1",`
2. Click on the sharing dialog
3. The sharing link is wrong, it's missing the `vhost1 string`

![image](https://user-images.githubusercontent.com/2115696/185558678-654728c9-3f95-41f7-aa2e-f11296b93df0.png)

## Edition and Platform
 - Edition: Personal Server
 - Version: 7.2.0

Add any other context about the problem here, and any notes about the severity: Sev 2

**Comment by wuwinson** [comment]
@Rajat-Dabade Can you help investigate this one as well? Thanks! cc @sbishel

**Comment by sbishel** [comment]
@harshilsharma63 - Could you work with @Rajat-Dabade to get his environment setup for this. As I recall there are some tricks to duplicating this environment.

**Comment by harshilsharma63** [comment]
@sbishel yeah, you need to run Nginx  to allow opening app on a subpath. @Rajat-Dabade lets catch up on this.
